### PR TITLE
feat: add code examples for help key

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2903/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/2903/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2903/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2903/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2903/playground/)

File: libs/components/indicators/testing/src/status-indicator/status-indicator-harness.spec.ts [hunk 1]
```diff
@@ -36,16 +39,18 @@ class TestComponent {
   public indicatorType = 'warning';
   public customDescription: string | undefined;
   public descriptionType: SkyIndicatorDescriptionType = 'warning';
+  public helpPopoverContent = '';
+  public helpPopoverTitle = '';
 }
 
-describe('Status indicator harness', () => {
+fdescribe('Status indicator harness', () => {
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/testing/src/status-indicator/status-indicator-harness.spec.ts]: Uh oh.

File: apps/code-examples/src/app/code-examples/datetime/date-range-picker/help-key/demo.component.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<div class="sky-padding-even-lg">
+  <form [formGroup]="formGroup">
+    <div class="sky-margin-stacked-lg">
```
Comment (Blackbaud-TrevorBurch) [inline apps/code-examples/src/app/code-examples/datetime/date-range-picker/help-key/demo.component.html]: Is this spacing needed.

File: apps/code-examples/src/app/code-examples/forms/file-drop/help-key/demo.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    TestBed.configureTestingModule({
+      imports: [DemoComponent, SkyHelpTestingModule],
+    });
+  });
+
+  it('should have the correct help key', async () => {
+    const { fixture, helpController } = await setupTest();
+
+    const helpButton = fixture.nativeElement.querySelector(
+      'sky-help-inline button',
```
Comment (Blackbaud-TrevorBurch) [inline apps/code-examples/src/app/code-examples/forms/file-drop/help-key/demo.component.spec.ts]: We should not add spec files for components which do not yet have a test harness.

**Review by Blackbaud-TrevorBurch — DISMISSED** [review]

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**